Firmware/ultralcd.cpp

```diff
@@ -4856,28 +4854,27 @@ static void mmu_loading_test_menu() {
 /// @param unloadLength Retract distance for removal (manual reload)
 void unload_filament(float unloadLength)
 {
-	custom_message_type = CustomMsg::FilamentLoading;
-	lcd_setstatuspgm(_T(MSG_UNLOADING_FILAMENT));
+    custom_message_type = CustomMsg::FilamentLoading;
+    lcd_setstatuspgm(_T(MSG_UNLOADING_FILAMENT));
 
     FSensorBlockRunout fsBlockRunout;
 
-    // Retract filament
```

Comment:
Why hard code it?

Reply:
Added a new define `FILAMENT_UNLOAD_FAST_RETRACT_LENGTH` with [3431d23](https://github.com/prusa3d/Prusa-Firmware/pull/4193/commits/3431d23df57f0dbbe5f1e7a60749dfefd33e84d3)